src/app/meetups/add-meetups/meetups-add.component.ts

```diff
@@ -115,4 +127,9 @@ export class MeetupsAddComponent implements OnInit {
     this.router.navigate([ '/meetups' ]);
   }
 
+  getTimestamp(date) {
+    const myDate = date.split('-');
+    return new Date(Date.UTC(myDate[0], myDate[1] - 1, myDate[2])).getTime();

```

Comment:
Date.UTC is not required as user will not evaluate UTC time, it is always locale time